jib-gradle-plugin/CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ All notable changes to this project will be documented in this file.
 
 ### Added
 
+- Added json output to file of image reference information after build. Defaults to `build/jib-image.json`, configurable with `jib.outputPaths.imageJson`. ([#2227](https://github.com/GoogleContainerTools/jib/pull/2227))
```

Comment:
I can't parse this.

I didn't look into the PR. Does the JSON record only the image reference? And no tags?

Reply:
no, my wording is awful, lemme fix this whole thing.